database/database.go

```diff
@@ -0,0 +1,104 @@
+package database
+
+import (
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"time"
+
+	"github.com/josephroberts/edge-node-manager/config"
+
+	"log"
+)
+
+func Start() error {
+	err := exec.Command(os.Getenv("GOPATH")+"/bin/tiedot", "-mode=httpd", "-dir="+filepath.Join(config.Persistant, config.Db), "-port="+config.DbPort).Start()
```

Comment:
This is a real mess of a line, hard to read and it's combining calls with parameters. I'd prefer to see this separated out into a function. Is there no alternative but to use an exec to start 'tiedot'?